packages/components/side_nav/src/side_nav.tsx

```diff
@@ -0,0 +1,25 @@
+import { Grid } from "@buoysoftware/anchor-layout";
+import { SideNavLogo } from "./side_nav_logo";
+
+interface OwnProps {
+  children?: React.ReactNode;
+}
+
+type SideNavProps = OwnProps;
+
+export const SideNav: React.FC<SideNavProps> = ({
+  children,
+}): React.ReactElement => {
+  return (
+    <Grid
+      gridTemplateRows="auto 1fr auto"
```

Comment:
I think you may be better served with grid template areas so that you can slot your child component into the grid by name e.g. side-nav-logo, side-nav-footer, side-nav-body then you can have
```
gridTemplateAreas="
  'side-nav-logo'
  'side-nav-body'
  'side-nav-footer'
"
```
(not 100% sure how you would write this as a styledProp but it's probably something like the above)

If you really wanted to go big, you could even make child utility components for the side_nav that limit the gridArea prop to values found in the grid template areas. Or simply set the gridArea on your other nav components like the footer.

If you're not too familiar with the technique this is pretty helpful: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Grid_Template_Areas Lemme know if you want to discuss it at all.

Reply:
Glad you brought this up. I had originally planned on grid template but had some trouble. Will look at this!